github/projects.go

```diff
@@ -267,6 +267,9 @@ type ProjectCard struct {
 	Note       *string    `json:"note,omitempty"`
 	CreatedAt  *Timestamp `json:"created_at,omitempty"`
 	UpdatedAt  *Timestamp `json:"updated_at,omitempty"`
+	Url        *string    `json:"url,omitempty"`
```

Comment:
Also, let's be consistent with https://developer.github.com/v3/projects/cards/#get-a-project-card example and place `URL` on top of `ColumnURL`, so it's next to other URL-related fields.